fatiando/mesher/geometry.py

```diff
@@ -92,6 +94,57 @@ def x(self):
     def y(self):
         return self.vertices[:, 1]
 
+    @property
+    def orientation(self):
+        area = self.get_area(absolute=False)
+        if area < 0:
+            return "CW"
+        else:
+            return "CCW"
+
+    def set_clockwise(self):
+        self.set_orientation("CW")
```

Comment:
This method is kind of unnecessary. Just having `set_orientation` is probably enough since this doesn't really save a lot of typing.